daemon/containerd/image_tag.go

```diff
@@ -42,8 +50,8 @@ func (i *ImageService) TagImage(ctx context.Context, imageID image.ID, newTag re
 
 		// Check if image we would replace already resolves to the same target.
 		// No need to do anything.
-		if replacedImg.Target.Digest == targetImage.Target.Digest {
-			i.LogImageEvent(imageID.String(), reference.FamiliarString(newTag), events.ActionTag)
+		if replacedImg.Target.Digest == newImg.Target.Digest {
+			i.LogImageEvent(newImg.Name, imageFamiliarName(newImg), events.ActionTag)
```

Comment:
This is changing `id` to `name`, correct? Mostly considering here that name could be easier to read, but `ID` may work also in case where there's no name associated. I recall we had some similar discussion around "network" events (I think) at some point, where we could not add a name to all of the related events, which made it more difficult to correlate events.

Reply:
Ah yes, good catch, this should be the ID